### PR TITLE
User Code

### PR DESCRIPTION
**Pull request by Theys96**

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -47,7 +58,7 @@ router.post("/register", function(req,res){
 			} else{
 				//Here we generate a random seed for encrypting, saved in user.seed. Likewise a code for the qrcode.
 				var seed = crypto.randomBytes(32).toString('hex');
-				var code = crypto.randomBytes(9).toString('base64');
+				var code = crypto.randomBytes(9).toString('base64').replace(/\+/g, 'a').replace(/\//g, 'Q');
```
Comment (Matt1994) [inline routes/users.js]: Why do we still need this line?

**Review by Matt1994 — COMMENTED** [review]
Overall looks fine. Just check my single comment

File: routes/users.js (hunk 1 above)
Comment (Theys96) [reply routes/users.js]: We totally don't, good point.

**Review by Matt1994 — APPROVED** [review]
Perfect. All good now

**Pull request merged**